Makefile

```
@@ -0,0 +1,21 @@
+.PHONY: clean timestamp setup
+
+all: setup build
+
+build: timestamp firmware/$$(TIMESTAMP)-left.uf2 firmware/$$(TIMESTAMP)-right.uf2
+
+clean:
+	rm ./firmware/*.uf2
+
+firmware/%-left.uf2 firmware/%-right.uf2: config/adv360.keymap timestamp
+	docker run --rm -it --name zmk \
+		-v $(PWD)/firmware:/app/firmware \
+		-v $(PWD)/config:/app/config:ro \
+		-e TIMESTAMP=$(TIMESTAMP) \
+		zmk
+
+setup:
+	docker build --tag zmk .
```

Even though it's phony, make sense to depend on `Dockerfile`, `config/west.yml`, and `bin/build.sh`? e.g.,

```suggestion
setup: Dockerfile bin/build.sh config/west.yml
	docker build --tag zmk --file Dockerfile .
```

---

+1